docs/docs/installation_guide/installation/pre-installation/instance-req.md

```diff
@@ -74,10 +74,15 @@ To add a role to an Amazon EC2 instance that is already running:
 ```
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
 source ~/.bashrc
-nvm install 12
-npm install -g serverless pnpm hygen
+nvm install v14.17.0
```

Comment:
always installing latest node v14 is better

Reply:
Sure, changed to v14